doc/api/test.md

```diff
@@ -2850,6 +2850,7 @@ The corresponding declaration ordered events are `'test:pass'` and `'test:fail'`
     `undefined` if the test was run through the REPL.
   * `name` {string} The test name.
   * `nesting` {number} The nesting level of the test.
+  * `type` {string} The test type.
```

Comment:
I don't think these docs are currently descriptive enough. Since there are only two possible values, maybe we should list them out.

Reply:
I updated the doc, but I had some trouble thinking of how to phrase it.